### ISSUE TITLE
urls not appearing for found Mypy notes

### ISSUE THREAD

**Issue by eroller** [leinardi/mypy-pycharm#69]
### Step 1: Are you in the right place?
- [X ] I have verified there are no duplicate active or recent bugs, questions, or requests
- [X ] I have verified that I am using the latest version of the plugin.

### Step 2: Describe your environment
 - Plugin version: `0.11.2`
 - PyCharm/IDEA version: `Professional 2020.2`
 - Mypy version: `0.790`

#### Steps to reproduce:

  1. Run scan on file that generates a Note containing http url (e.g. `http://mypy.readthedocs.io/en/latest/common_issues.html#variance`)

#### Observed Results:

  * The note just says `"Dict" is invariant -- See http`
  

#### Expected Results:

  * The note should include the url from mypy, for example `"Dict" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance` 

#### Relevant Code:

```python
class A: ...
class B(A): ...

lst = [A(), A()]  # Inferred type is List[A]
new_lst = [B(), B()]  # inferred type is List[B]
lst = new_lst  # mypy will complain about this, because List is invariant
```

**Comment by eroller** [comment]
looks like some kind of truncation is happening?

**Comment by jaredtkatz** [comment]
yeah, a lot of messages seem to get truncated in the extension toolbar and the mouseover tooltip, which really undermines the usefulness of this plugin. Feel like this must be an easy bug to fix...
